.gitignore

```diff
@@ -10,7 +10,6 @@ local.properties
 *.*~
 
 # gradle env props
-gradle.properties

```

Comment:
Please revert this change.

People have different configurations for their local machine which shouldn't be shared forcefully with others.

Reply:
yeah I have added gradle.properties to .gitignore file

Reply:
But the file gradle.properties is still showing up. That needs to be removed too

Reply:
I have deleted the gradle.properties file